src/common/components/ui/EventsModal.css

```diff
@@ -0,0 +1,8 @@
+.events-modal .pf-c-modal-box__body {
+  padding-top:0;
+  display: flex;
+  flex-direction: column;
+}
+.events-modal .events-list-wrapper {
```

Comment:
I'd rather see that the `events-list-wrapper` style will be passed as className prop to `EventListFetch` and ends up applied to `EventsList` table component.

And according to BEM it would be named something like `events-modal__event-list`

Reply:
The table needs a wrapper (I have renamed it to `events-modal__event-list`) otherwise it doesn't scroll. But everything else I was able to implement, I hope.